src/main/java/com/machowina/exception/RestResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.machowina.exception;
+
+import java.time.format.DateTimeParseException;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class RestResponseEntityExceptionHandler extends ResponseEntityExceptionHandler {
+	
+	
+	@ExceptionHandler(EntityNotFoundException.class)
+    @ResponseStatus(value=HttpStatus.NOT_FOUND)
+    @ResponseBody
+    public String handleEntityNotFoundException(HttpServletRequest request, EntityNotFoundException ex) {
+        
+        return request.getRequestURL().toString() + " - " + ex.getMessage();
```

Comment:
czy użytkownik potrzebuję informacji na jaki URL przed chwilą uderzył?

Reply:
Założyłam że jeśli daję wiadomość "nie ma samochodu o takim id" to dobrze dołączyć o jakie id chodzilo. 
Ale może trochę przesada